go/pkg/client/tree.go

```diff
@@ -131,9 +132,36 @@ func getTargetRelPath(execRoot, path string, symMeta *filemetadata.SymlinkMetada
 	return relExecRoot, relSymlinkDir, err
 }
 
+// getRemotePath returns a remote path by transforming the local input path
+// in the transformed path workingDir component is replaced with remoteWorkingDir
```

Comment:
Can you rephrase this comment? I think its supposed to be two separate sentences maybe?

Reply:
addressed in the latest commit